libs/vkd3d/resource.c

```diff
@@ -951,9 +951,19 @@ static HRESULT vkd3d_get_image_create_info(struct d3d12_device *device,
         uint32_t candidate_alignment = desc->Alignment ?
                 desc->Alignment : D3D12_DEFAULT_RESOURCE_PLACEMENT_ALIGNMENT;
 
-        /* Only consider alignments that are <= to the requested alignment. */
-        while (candidate_alignment && !(candidate_alignment & supported_alignment))
-            candidate_alignment >>= 1;
+        if (desc->Flags & (D3D12_RESOURCE_FLAG_ALLOW_RENDER_TARGET | D3D12_RESOURCE_FLAG_ALLOW_DEPTH_STENCIL))
```

Comment:
I'm a little wary about making this the default when we don't fully understand the performance implication. Would it be possible to make it a config flag first? If we run into any such cases in the wild we can re-evaluate.